oci/build-image-manifest.py

```diff
@@ -25,9 +25,12 @@ def hash(data):
 
     non_empty_layers = []
     for layer in opts.layers:
-        if os.path.exists(os.path.join(layer, 'layer.tar')):
```

Comment:
The layer could have a different suffix so we just check for `metadata.json` instead